acsoo/addons.py

```diff
@@ -85,3 +84,102 @@ def addons_list_depends(ctx, exclude):
 
 
 addons.add_command(addons_list_depends, 'list-depends')
+
+
+@click.command(help="Print a comma separated list of the modified installable "
+                    "addons from the given git ref")
+@click.argument('git_ref')
+@click.option('-d', '--tmp-dir', 'tmp_dir', default='tmp_acsoo_toupdate',
+              type=click.Path(file_okay=False),
+              help='Path where the repository will be cloned for comparison')
```

Comment:
No need to specify a temp dir. Use [tempfile.mkdtemp](https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp)